packages/core/src/lib/markdown-it/plugins/markdown-it-block-embed/services/VideoServiceBase.js

```diff
@@ -46,24 +46,22 @@ class VideoServiceBase {
     let escapedServiceName = this.env.md.utils.escapeHtml(this.name);
     containerClassNames.push(this.env.options.serviceClassPrefix + escapedServiceName);
 
+    let containerStyles = [];
+    containerStyles.push(["position: relative;"]);
+    if (this.options.height !== undefined && this.options.width !== undefined) {
+      containerStyles.push(`padding-bottom: ${this.options.height / this.options.width * 100}%`);
+    }
+
     let iframeAttributeList = [];
     iframeAttributeList.push([ "type", "text/html" ]);
     iframeAttributeList.push([ "src", this.getFilteredVideoUrl(videoID) ]);
-    iframeAttributeList.push([ "frameborder", 0 ]);
-
-    if (this.env.options.outputPlayerSize === true) {
-      if (this.options.width !== undefined && this.options.width !== null) {
-        iframeAttributeList.push([ "width", this.options.width ]);
-      }
-      if (this.options.height !== undefined && this.options.height !== null) {
-        iframeAttributeList.push([ "height", this.options.height ]);
-      }
-    }
+    iframeAttributeList.push([ "scrolling", "yes" ]);
```

Comment:
Missed out a leftover from @tlylt's suggestion, cmiiw.

One more strange empty line below I think we could remove too.

Reply:
yep, i missed this removing now! thanks